ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectDataTranslator.java

```diff
@@ -231,7 +232,7 @@ private Object toKsqlValue(
       case STRUCT:
         return toKsqlStruct(schema, connectSchema, (Struct) convertedValue, pathStr);
       case STRING:
-        // use String.valueOf to convert various int types and Boolean to string
+        // use String.valueOf to convert various int types, Struct and Boolean to string
         return String.valueOf(convertedValue);
```

Comment:
I think something wrong is gonna happen if we done here. The ConnectDataTranslator is a generic class for all Connect types (avro, protobuf, json). The struct->string conversion should be in JSON string, i.e. `{k1=v1}`. You can verify this with a plain JSON format or with JSON_SR in version 0.26 or lower. I actually not sure what Avro/Protobuf should do wit Struct->String. 

1. Could you check in older ksql versions if Struct->String is supported with Avro and Protobuf?
2. Could you check in older ksql versions what Struct->String wit JSON_SR and plain JSON display?

Reply:
> I think something wrong is gonna happen if we done here.
The issue addresses exactly this line for change (please have a look at the issue explanation)